packages/babel-types/test/builders/es2015/templateElement.js

```diff
@@ -21,5 +21,31 @@ describe("builders", function() {
         expect(() => t.templateElement("foo")).toThrowErrorMatchingSnapshot();
       });
     });
+    describe("templateLiteral", function() {
+      it("should validate", function() {
+        const foo = t.templateElement({ raw: "foo" });
+        const bar = t.templateElement({ raw: "bar" });
+
+        expect(() => t.templateLiteral([foo], [])).toMatchSnapshot();
+
+        expect(() => t.templateLiteral([foo, bar], ["baz"])).toMatchSnapshot();
+
+        expect(() =>
+          t.templateLiteral([foo, bar], ["baz", "qux"]),
+        ).toThrowErrorMatchingSnapshot();
+
+        expect(() =>
+          t.templateLiteral([foo, bar], []),
+        ).toThrowErrorMatchingSnapshot();
+
+        expect(() =>
+          t.templateLiteral({}, ["baz"]),
```

Comment:
The first needs to be an array, and the second an array of expressions (not a string).

Reply:
Those tests are put there intentionally to show that the previous validators still trigger and that the only time the new validation rule triggers is when that's the only thing wrong.

I can remove it though if it's considered irrelevant.